kubernetes/client/apis/admissionregistration_v1alpha1_api.py

```diff
@@ -268,18 +268,18 @@ def delete_collection_initializer_configuration_with_http_info(self, **kwargs):
                                         _request_timeout=params.get('_request_timeout'),
                                         collection_formats=collection_formats)
 
-    def delete_initializer_configuration(self, name, body, **kwargs):
```

Comment:
This looks like it’s potentially going to break some codes if they called the function with func(name, body). Now user needs to call with func(name, body=xxx)

Reply:
Looks like all delete methods will have this change. Probably want to document this

Reply:
It was documented in API change. I added a duplicate of the body param changelog in Breaking Change.